app/plugins/placeholdifier.js

```diff
@@ -0,0 +1,23 @@
+import { loadStyles } from '../utilities/styles.js'
+
+export const commands = [
+    'placeholdifier',
+]
+
+export const description = 'turn the page into a live wireframe'
+
+export default async function() {
+    const stylesheet = document.createElement('link')
+    stylesheet.setAttribute('rel', 'stylesheet')
+    stylesheet.setAttribute('href', 'https://unpkg.com/placeholdifier/placeholdifier.css')
```

Comment:
```suggestion
    stylesheet.setAttribute('href', 'https://unpkg.com/placeholdifier/placeholdifier.css')
    document.head.appendChild(stylesheet)
```

i must have forgotten to add this line that appends the created link tag to the page

Reply:
yes, good catch